microbench/src/main/java/io/netty/microbench/handler/ssl/AbstractSslEngineThroughputBenchmark.java

```diff
@@ -66,14 +66,16 @@ public final void tearDown() throws Exception {
 
     protected void doTearDown() throws Exception { }
 
-    protected final ByteBuffer doWrap() throws SSLException {
-        wrapSrcBuffer.position(0).limit(messageSize);
-
+    protected final ByteBuffer doWrap(int numWraps) throws SSLException {
         wrapDstBuffer.clear();
 
-        SSLEngineResult wrapResult = clientEngine.wrap(wrapSrcBuffer, wrapDstBuffer);
+        for (int i = 0; i < numWraps; ++i) {
+            wrapSrcBuffer.position(0).limit(messageSize);
 
-        assert checkSslEngineResult(wrapResult, wrapSrcBuffer, wrapDstBuffer);
+            SSLEngineResult wrapResult = clientEngine.wrap(wrapSrcBuffer, wrapDstBuffer);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "wrapResult". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1854)